app-editors/vscode/vscode-1.59.1.ebuild

```diff
@@ -88,6 +88,7 @@ src_install() {
 	doins -r *
 	fperms +x /opt/${PN}/{,bin/}code
 	fperms 4711 /opt/${PN}/chrome-sandbox
+	fperms 755 /opt/${PN}/resources/app/extensions/git/dist/askpass*.sh
```

Comment:
The glob `*` doesn't work with `fperms` :\ 

it passes if you have the files already installed, but on new installs it fails with:
```
>>> Install app-editors/vscode-1.59.1 into /var/tmp/portage/app-editors/vscode-1.59.1/image
chmod: cannot access '/var/tmp/portage/app-editors/vscode-1.59.1/image/opt/vscode/resources/app/extensions/git/dist/askpass*.sh': No such file or directory
 * ERROR: app-editors/vscode-1.59.1::gentoo failed (install phase):
 *   fperms failed
```
been meaning to try and fix fperms for that but haven't looked it yet.

Same thing with vscodium.

Reply:
Alright thanks! let me fix that.